docs/source/configuration.rst

```diff
@@ -1,44 +1,72 @@
 Configuration
 =============
 
+UCX can be configured with a wide variety of options and optimizations including: transport, caching, etc.
+
+For novice user we recommend the following settings:
+
+::
+
+    UCX_MEMTYPE_CACHE=n UCX_TLS=all
+
+``UCX_TLS=all`` instructs UCX to try all available transport methods.  However, users who want to define what transport methods and/or other optional settings may do so.  Below we define the more common environment variables and provide some example combinations and usage.
+
 Env Vars
-----------
+--------
 
 DEBUG
 ~~~~~
 
+Debug variables for both UCX and UCX-PY can be set
 
-``UCX_PY_LOG_LEVEL``
+``UCXPY_LOG_LEVEL``
 ``UCX_LOG_LEVEL``
 
 Values: DEBUG, TRACE
 
+
 MEMORY
 ~~~~~~
 
 ``UCX_MEMTYPE_CACHE``
 
-UCX Memory optimization `known issues <https://github.com/openucx/ucx/wiki/NVIDIA-GPU-Support#known-issues>`_.  UCX-PY regularly sets this to `n` -- toggles whether UCX library intercepts cu*alloc* calls.
+This is UCX Memory optimization which toggles whether UCX library intercepts cu*alloc* calls.  UCX-PY defaults this value to  ``n``.  There `known issues <https://github.com/openucx/ucx/wiki/NVIDIA-GPU-Support#known-issues>`_ when using this feature.
 
 Values: n
 
-```UCX_RNDV_SCHEME``
+``UCX_RNDV_SCHEME``
 
-Values: put_zcopy
+Communication scheme in RNDV protocol
+
+Values:
+
+- ``put_zcopy``
+- ``get_zcopy``
+- ``auto`` (default)
 
 
 ``UCX_TLS``
 
-Values:
-- rc = ibv_post_send, ibv_post_recv, ibv_poll_cq
-- cuda_copy = cuMemHostRegister, cuMemcpyAsync
-- cuda_ipc =  cuIpcCloseMemHandle ,  cuIpcOpenMemHandle, cuMemcpyAsync
-- sockcm = connection management over sockets
-- tcp = communication over TCP
+Transport Methods (Simplified):
+
+- ``rc`` -> (IB) ibv_post_send, ibv_post_recv, ibv_poll_cq
+- ``cuda_copy`` -> cuMemHostRegister, cuMemcpyAsync
+- ``cuda_ipc`` -> (NVLINK) cuIpcCloseMemHandle, cuIpcOpenMemHandle, cuMemcpyAsync
```

Comment:
Might be worth stating CUDA Interprocess Communication explicitly and then relegating the functions to a parenthesized list afterwards.